include/sys/refcount.h

```diff
@@ -63,57 +63,57 @@ typedef struct refcount {
  * refcount_create[_untracked]()
  */
 
-void refcount_create(zfs_refcount_t *rc);
-void refcount_create_untracked(zfs_refcount_t *rc);
-void refcount_create_tracked(zfs_refcount_t *rc);
-void refcount_destroy(zfs_refcount_t *rc);
-void refcount_destroy_many(zfs_refcount_t *rc, uint64_t number);
-int refcount_is_zero(zfs_refcount_t *rc);
-int64_t refcount_count(zfs_refcount_t *rc);
+void zfs_refcount_create(zfs_refcount_t *rc);
+void zfs_refcount_create_untracked(zfs_refcount_t *rc);
+void zfs_refcount_create_tracked(zfs_refcount_t *rc);
```

Comment:
While you're here, if you'd like to remove the argument names (throughout this header file), that would be good too.

Reply:
Sure, I can do that. It will have to wait until afternoon though (in about 7 hours).